script/build

```diff
@@ -1,7 +1,22 @@
 #!/bin/bash
-mkdir -p build
```

Comment:
There was an issue for when `gox` is not installed it would continue and perform the subsequent commands. It could have been fixed without rewriting it but I like to follow [defensive bash programming](http://www.kfirlavi.com/blog/2012/11/14/defensive-bash-programming/). If you would like I can fix it without a rewriting the file.